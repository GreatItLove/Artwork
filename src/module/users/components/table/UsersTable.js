import React from 'react';

import HeaderRightActionIcons from '../../containers/HeaderRightActionIcons';
import Panel from '../../../common/components/Panel';
import SystemMessageOnUserList from '../../containers/SystemMessageOnUserList';

import UsersList from '../../containers/UsersList';
import Refresh from '../../containers/RefreshUsers';

/**
 * @function UsersTable
 * @returns {XML}
 */
function UsersTable() {
  return (
    <div>
      <SystemMessageOnUserList />
      <Panel
        title="Users"
        refresh={<Refresh />}
        rightElement={<HeaderRightActionIcons />}>
        <UsersList />
      </Panel>
    </div>
  );
}

export default UsersTable;
