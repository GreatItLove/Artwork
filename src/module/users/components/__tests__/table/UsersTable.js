import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import UsersTable from '../../table/UsersTable';

describe('UsersTable', () => {
  it('Snapshot of UsersTable', () => {
    const wrapper = shallow(<UsersTable />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
