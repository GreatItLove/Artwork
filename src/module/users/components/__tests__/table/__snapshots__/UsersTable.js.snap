// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UsersTable Snapshot of UsersTable 1`] = `
<div>
  <Connect(SystemMessageOnUserList) />
  <Component
    rightElement={<Connect(HeaderRightActionIcons) />}
    title="Users"
  >
    <Translate(Connect(FetchUsersIfNotExist)) />
  </Component>
</div>
`;
